CONTRIBUTING.md

```diff
@@ -5,9 +5,17 @@
 * Use `use` statements wherever possible instead of writing the fully qualified name.
 * Order the composer/npm dependencies alphabetically.
 * Do not use code from the [includes](./includes) directory anywhere else.
-* Please cover your code by unit tests. Code under `includes` does not require tests.
+* Please cover your code by unit tests, our goal is 100% line coverage. Code under `includes` does not require tests as it's mostly not testable.
```

Comment:
Right now, it is not only our goal to have 100% line coverage in the new source folder, but it is required to get the PR merged. May we should make that more clear?